spec/rubocop/cop/lint/literal_as_condition_spec.rb

```diff
@@ -9,6 +9,10 @@
           top
         end
       RUBY
+
+      expect_correction(<<~RUBY)
```

Comment:
Let's add `expect_no_corrections` to the other cases to confirm nothing unexpected is going on.